learn/documentation.adoc

```diff
@@ -30,6 +30,8 @@
 
 '''
 
+////
+
```

Comment:
I 'd rather not see them commented out - we need to people to know that we're doing non final releases too (even if they visit us in the period between Final and Beta1).